napari/layers/points/points.py

```diff
@@ -379,15 +379,19 @@ def __init__(
             continuous_colormap=edge_colormap,
             contrast_limits=edge_contrast_limits,
             categorical_colormap=edge_color_cycle,
-            properties=properties,
+            properties=self._properties
+            if self._data.size
+            else self._property_choices,
```

Comment:
Is black really happy with this??? I am surprised it doesn't need to be indented...

Reply:
this is black output. Before black it was one line.